packages/physics-lite/src/shape/LiteBoxColliderShape.ts

```diff
@@ -55,6 +57,48 @@ export class LiteBoxColliderShape extends LiteColliderShape implements IBoxColli
     this._setBondingBox();
   }
 
+  /**
+   * {@inheritDoc IColliderShape.pointDistance }
+   */
+  override pointDistance(position: Vector3, rotation: Quaternion, point: Vector3): IPointDistanceInfo {
```

Comment:
why this api is not same as physics's `getDistanceAndClosestPointFromPoint`

Reply:
pointDistance is  API name of physX, we can't pass pointer to native